pkg/controller/smskeys/render.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package smskeys
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/google/exposure-notifications-verification-server/pkg/controller"
+	"github.com/google/exposure-notifications-verification-server/pkg/database"
+	"github.com/google/exposure-notifications-verification-server/pkg/keyutils"
+)
+
+func (c *Controller) redirectShow(ctx context.Context, w http.ResponseWriter, r *http.Request) {
+	http.Redirect(w, r, "/realm/smskeys", http.StatusSeeOther)
+}
+
+func (c *Controller) renderShow(ctx context.Context, w http.ResponseWriter, r *http.Request, realm *database.Realm) {
+	m := controller.TemplateMapFromContext(ctx)
+	m.Title("Authenticated SMS Signing keys")
+	m["realm"] = realm
+
+	keys, err := realm.ListSMSSigningKeys(c.db)
```

Comment:
This should be cached (if we don't have a ListSMSSigningKeysCached, we should make one)

Reply:
I don't think we want to cache in this instance - this is the admin screen.
We do cache the public keys, so we're not hitting KMS on every refresh.